README.md

```diff
@@ -76,6 +78,37 @@ We recommend installing RDKit with deepchem if you use conda.
 conda install -y -c conda-forge rdkit
 ```
 
+### Latest version

```

Comment:
Let's just name this section "Nightly build version" and remove the old directions for nightly build version below